rust/ql/lib/codeql/rust/elements/internal/LabelableExprImpl.qll

```diff
@@ -0,0 +1,30 @@
+/**
+ * This module provides a hand-modifiable wrapper around the generated class `LabelableExpr`.
+ *
+ * INTERNAL: Do not use.
+ */
+
+private import codeql.rust.elements.internal.generated.LabelableExpr
+
+/**
+ * INTERNAL: This module contains the customizable definition of `LabelableExpr` and should not
+ * be referenced directly.
+ */
+module Impl {
+  // the following QLdoc is generated: if you need to edit it, do it in the schema file
+  /**
+   * The base class for expressions that can be labeled (`LoopExpr`, `ForExpr`, `WhileExpr` or `BlockExpr`).
+   */
+  class LabelableExpr extends Generated::LabelableExpr {
+    override string toString() {
+      result = concat([this.getLabel().toString() + ":", this.toStringPrefix(), "{ ... }"], " ")
```

Comment:
Recursion.

Reply:
Also, needs to be ordered (e.g. `concat(["a", "c", "b"])` yields `"abc"` and not `"acb"`).

Reply:
uh, forgot this one when I fixed that in other parts